packages/vitest/src/defaults.ts

```diff
@@ -96,4 +96,4 @@ const config = {
   slowTestThreshold: 300,
 }
 
-export const configDefaults: Required<Pick<UserConfig, keyof typeof config>> = Object.freeze(config)
+export const configDefaults = Object.freeze(config as Required<UserConfig>)
```

Comment:
This actually breaks `configDefaults` types. It should ONLY reference keys that exist in default config

Reply:
I cannot review it rn, I will try to fix it tmr...